test/database/apikeys_test.go

```diff
@@ -0,0 +1,151 @@
+package database
+
+import (
+	"context"
+	"testing"
+
+	"github.com/SkynetLabs/skynet-accounts/database"
+	"github.com/SkynetLabs/skynet-accounts/test"
+)
+
+// TestAPIKeys ensures the DB operations with API keys work as expected.
+func TestAPIKeys(t *testing.T) {
+	t.Parallel()
+
+	ctx := context.Background()
+	dbName := test.DBNameForTest(t.Name())
+	db, err := database.NewCustomDB(ctx, dbName, test.DBTestCredentials(), nil)
```

Comment:
I like your `test` package. I think it could be better though if you'd make it `test.NewDatabase()` and then also just handle the `dbName` thing internally, so it'd be `test.NewDatabase(ctx, t.Name(), nil)` that would be pretty epic and then I'll copy it 😛